docs/runbooks/SSPTemplateValidatorDown.md

```diff
@@ -1,30 +1,42 @@
 # SSPTemplateValidatorDown
+<!--apinnick, Nov 2022-->
 
 ## Meaning
 
-The Template Validator is responsible for validating the Virtual Machines do not violate the templates they are assigned to.
-This alert fires when all the Template Validator's pods are down.
+This alert fires when all the Template Validator pods are down.
+
+The Template Validator checks virtual machines (VMs) to ensure that they do not violate their templates.
 
 ## Impact
 
-With all Template Validator's pods down the Vms will not be validated against the templates they are associated with.
+VMs are not validated against their templates.
```

Comment:
@borod108 I think we are missing the impact of what will happen if VMs are not validated against their templates.

Reply:
Good point. We can add something like:
This means VMs may be created with specifications that do not much their respective workloads.